components/SectionHero.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+    <section class="mt-10 lg:mt-[174px]">
+        <div class="">
+            <div class="container">
+                <h1 class=" text-3xl md:text-5xl lg:text-[86px] text-white text-center  lg:!leading-[96px]">
+                    PORTA IL TUO
+                    <span class="italic font-medium relative">
+                        PADEL CLUB
+                        <span
+                            class=" bg-secondary text-app-bg text-2xl px-4 py-2.5 -top-8 -right-12 hidden md:inline absolute transform rotate-[13.9deg]  my-2 lg:my-0">A
+                            COSTO
+                            ZERO</span>
+                    </span> <br>
+                    AL
+                    <span class=" italic font-medium">
+                        LIVELLO SUCCESSIVO
+                    </span>
+                </h1>
+                <p class="text-white text-center text-lg md:text-xl lg:text-3xl mt-6">
+                    Aumenta gli accessi, ospita gli eventi esclusivi e i nostri viaggi <br> allenamento. Ci occuperemo
+                    di
+                    supportarti nell’intera organizzazione.
+                </p>
+                <div class="mt-11 flex items-center justify-center">
+                    <button class="btn-primary--large">
+                        Candidati subito
+                    </button>
+                </div>
+            </div>
+            <div class="flex items-center justify-center flex-wrap mt-12">
+                <img src="/hero-img-1.png" alt=" hero img-1" class="-mr-24 mt-20 z-30">
+
+                <img src="/hero-img-2.png" alt=" hero img-1" class="-mr-10 z-20">
+
+                <img src="/hero-img-3.png" alt=" hero img-1" class="-mr-16 mt-10 z-10">
+
+                <img src="/hero-img-4.png" alt=" hero img-1" class="">
```

Comment:
All these images on mobile breaks the page going too much on the right
![Screenshot 2024-05-21 alle 15 31 28](https://github.com/93lucasp/frontend-assessment-A/assets/12210564/8114b311-417e-4195-b5b5-f046ed868fe9)